packages/xstate-wallet/src/constants.ts

```diff
@@ -7,7 +8,14 @@ export const NITRO_ADJUDICATOR_ADDRESS =
   process.env.NITRO_ADJUDICATOR_ADDRESS || '0x0000000000000000000000000000000000000000';
 
 // TODO: Move top ENV variable
-export const HUB_ADDRESS = process.env.HUB_ADDRESS || '0x100063c326b27f78b2cBb7cd036B8ddE4d4FCa7C';
+// Corresponds to '0x8624ebe7364bb776f891ca339f0aaa820cc64cc9fca6a28eec71e6d8fc950f29' from test data
+export const HUB_ADDRESS = process.env.HUB_ADDRESS || '0xaaaa84838319627Fa056fC3FC29ab94d479B8502';
```

Comment:
It might be a good idea to add an `env.js` file, or `config.js` file (with a `Config` object)

Reply:
I'd like this as a separate PR